### ISSUE TITLE
BlankOn / BlankOff actions no response

### ISSUE THREAD

**Issue by logosway** [bitfocus/companion-module-vivitek-projector#4]
I am able to setup actions for Power On/Off. But when I setup a button for Blank On and Blank Off, the projector doesn't response (tried both "blank" and "shutter" actions). 
My projector model is DU8190Z. It responded when I enable blank through the web control only. Is there any setup I missed?
Attached my web control interface here.
![Capture](https://github.com/user-attachments/assets/5be3b1fc-e794-48a5-a01b-b171ba0a2030)

**Comment by WaffleTime16** [comment]
We have a DU9800Z and are experiencing the same fault.  Blank has no effect via Companion.

**Comment by WaffleTime16** [comment]
OK, was just able to hack together a bit of workaround using the projector's web interface.  This module uses direct TCP, but I was able to see what the web UI was sending to initiate the blank command and create a generic HTTP connection to send that GET request.  It'd be great if someone with more knowledge could patch this so it works inside the module, but for now this is a solution for us.

**Comment by logosway** [comment]
@WaffleTime16 Can you share a bit more how do you do that?

**Comment by WaffleTime16** [comment]
Certainly!  My understanding of HTTP requests is limited, but I was able to scrub my browser sending a GET request to the projector IP to a script called pjctl.cgi.  It added either shutterOpen=+ or shutterClose=+ and a date stamp.  

So in companion I created a connection using a generic HTTP module that points at the static IP for the projector and sends a toggled GET request to these URLs:

pjctl.cgi?shutterClose=+&CLK=$(date +%s)

pjctl.cgi?shutterOpen=+&CLK=$(date +%s)
